content/en/boilerplates/helm-preamble.md

```diff
@@ -5,4 +5,4 @@ in this guide are the same underlying charts used when
 installing Istio via [Istioctl](/docs/setup/install/istioctl/) or the
 [Operator](/docs/setup/install/operator/).
 
-This feature is currently considered [alpha](/docs/releases/feature-stages/).
```

Comment:
It's still alpha. Given the direction of where validating webhooks w/r/t revisions are we haven't pursued Helm being promoted to Beta just yet.

Reply:
boooo